extensions/adobe/experience/aam-experienceevent.schema.json

```diff
@@ -0,0 +1,53 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/aam-experienceevent",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Adobe Audience Manager Template Mixin",
+  "type": "object",
+  "description": "Adobe Audience Manager Mixin for use with Schemas for Solution data ingestion. Includes the core/standard ExperienceEvent as well as the other required Core mixins.",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "meta:extends": [
+    "https://ns.adobe.com/xdm/context/experienceevent-segmentmembership",
+    "https://ns.adobe.com/xdm/context/experienceevent-enduserids-deprecated",
+    "https://ns.adobe.com/xdm/context/device",
+    "https://ns.adobe.com/xdm/context/environment",
+    "https://ns.adobe.com/xdm/context/placecontext",
+    "https://ns.adobe.com/xdm/context/profileStitch"
```

Comment:
These last 4 includes are all data types, they do not introduce concrete fields.
You should include the equivalent https://ns.adobe.com/xdm/context/experienceevent-*

Reply:
Changed.